src/docs/uxd/toolbarsFilters/DOC.md

```diff
@@ -1,5 +1,102 @@
 # Toolbars & Filters
 
-Content TBD
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/toolbarLayout.png?raw=true "toolbarLayout")
 
-[Google Doc](https://docs.google.com/document/d/1Uq1cmLxrUBhhDZrUe7KIdesLKORT2IF5M13XqCJ6yYY/edit?usp=sharing)
+1. **Filtering**: Filtering should be on the left-most side of the toolbar. All lists should have some filtering. (See filtering section for more details)
+2. **Action buttons**: Arrange action buttons to the right of the filtering. There can be 1 primary action (an action is not required), 1 secondary action, and any overflow goes into the kebab.
+3. **Kebab & Export**: Export should always go into the kebab, even if it is the only action on the table or list.
+4. **Pagination**: Use pagination on top and bottom of table. [PF4 pagination](https://v2.patternfly.org/documentation/core/components/pagination) will responsively react to show only the results count when in smaller spaces. This is the only results count shown. (See filtering section for more details about how the results count works with filtering)
+5. **Export**: Export and additional icons go into the kebob icon.
+
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/toolbarExport.png?raw=true "toolbarExport")
+
+*Possible future work includes sticky headers.*
+This pattern is most similar to the "Toolbar w/ pagination" example here: https://v2.patternfly.org/documentation/core/components/toolbar (as of 4/2)
+
+## Filters
+
+The [latest release plan from Patternfly](https://docs.google.com/document/d/1xMQBWVX0RZqCv1N5hAH-82T78SrJxqSBu1ihF-ytqfI/edit#heading=h.pr9zj26f2jlv) has filters to be delivered early May.
```

Comment:
cut this whole thing because its full of lies and they aren't delivering anything around filters till august 😭